### PR TITLE
clean up BookMarkDirAdd doc and code

### PR DESCRIPTION
**Pull request by Konfekt**

### CODE REVIEW

**Comment by ludovicchabant** [comment]
Looks good.

Since `g:ctrlp_mruf_tilde_homedir` is also used in BookmarkDirs, and may also start creeping up in other places, would it make sense to rename it to `g:ctrlp_homedir_paths` or something?

**Comment by Konfekt** [comment]
Ok. Done! I kept the name as I found it more telling than `homedir_paths`.

**Comment by Konfekt** [comment]
By the way, any news about my other pull request? Feel free to readd the autocmd's I hastily removed.

File: autoload/ctrlp.vim [hunk 1]
```diff
@@ -218,6 +218,7 @@ fu! s:opts(...)
 	for each in ['byfname', 'regexp'] | if exists(each)
 		let s:{each} = {each}
 	en | endfo
+	if !exists('g:ctrlp_tilde_homedir') | let g:ctrlp_tilde_homedir = 0 | en
```
Comment (ludovicchabant) [inline autoload/ctrlp.vim]: shouldn't this actually be initialized around line 66 in that big array that copies global settings to script local variables?

**Comment by ludovicchabant** [comment]
Oh yeah your other pull request I wanted to test actually because all those Vim documentation files in my MRU list are driving me crazy :)

File: autoload/ctrlp.vim (hunk 1 above)
Comment (Konfekt) [inline autoload/ctrlp.vim]: Yeah I thought so, but it is never used in the file itself. In contrast to all those other variables. 

And surprisingly the arraybonly sets default values for the s:variables.

**Pull request merged**